app/components/verkiezingen/bcsd-voorzitter-alert.hbs

```diff
@@ -1,9 +1,13 @@
-<div {{did-insert (perform this.handleErrorMessage)}}>
-  {{#if this.errorMessage}}
-    <div class="au-o-box">
-      <AuAlert @skin="warning" @icon="alert-triangle" @closable={{true}}>
-        <p>{{this.errorMessage}}</p>
-      </AuAlert>
-    </div>
-  {{/if}}
-</div>
\ No newline at end of file
+{{#if (await @bestuursorgaanInTijd.isBCSD)}}
+  <div {{did-update this.mandatarissenUpdated @trigger}}>
+    {{#if this.messages}}
+      <div class="au-o-box">
+        <AuAlert @skin="warning" @icon="alert-triangle" @closable={{true}}>
+          {{#each this.messages as |warning|}}
+            <p> - {{warning.message}}</p>
```

Comment:
would remove the -, it looks silly when there is only one warning and if there are multiple, it doesn't hurt that there isn't any - in front of them